packages/compressed-textures/src/resources/BlobResource.ts

```diff
@@ -22,18 +22,19 @@ export abstract class BlobResource extends BufferResource
     protected loaded: boolean;
 
     /**
-     * @param {string} source - the URL of the texture file
+     * @param source - the buffer/URL of the texture file
      * @param {PIXI.IBlobOptions} options
      * @param {boolean}[options.autoLoad] - whether to fetch the data immediately;
      *  you can fetch it later via {@link PIXI.BlobResource#load}
      * @param {boolean}[options.width] - the width in pixels.
      * @param {boolean}[options.height] - the height in pixels.
      */
-    constructor(source: string | Uint8Array | Uint32Array | Float32Array,
-        options: IBlobOptions = { width: 1, height: 1, autoLoad: true })
+    constructor(source: string | Int8Array | Uint8Array | Uint8ClampedArray | Int16Array | Uint16Array | Int32Array
```

Comment:
should we consider making a type for :`Int8Array | Uint8Array | Uint8ClampedArray | Int16Array | Uint16Array | Int32Array | Uint32Array
        | Float32Array`

Might make all the functions params a little less wordy!

Reply:
Created the `BufferType` type. Not sure if it's the best name though. Maybe more specific like `BufferResourceDataType` is better?